### ISSUE TITLE
Error when importing `astropy.nddata.utils.block_reduce` 

### ISSUE THREAD

**Issue by ladsantos** [spacetelescope/stistools#143]
When using `astropy` v5.1 and I do a `import stistools` (v1.4.1), I get the following error:
```
ImportError                               Traceback (most recent call last)
/var/folders/0q/ph_vqx8j58s5t8b4smvzpcf80002vw/T/ipykernel_62037/474873755.py in <module>
      3 import matplotlib.pylab as pylab
      4 import matplotlib.colors as plc
----> 5 import stistools
      6 import astropy.units as u
      7 import astropy.constants as c

~/Applications/miniconda3/envs/dat/lib/python3.9/site-packages/stistools-1.4.1-py3.9.egg/stistools/__init__.py in <module>
     16 from . import tastis
     17 from . import ctestis
---> 18 from . import defringe
     19 
     20 # These lines allow TEAL to print out the names of TEAL-enabled tasks

~/Applications/miniconda3/envs/dat/lib/python3.9/site-packages/stistools-1.4.1-py3.9.egg/stistools/defringe/__init__.py in <module>
      1 from .prepspec import prepspec
      2 from .normspflat import normspflat
----> 3 from .mkfringeflat import mkfringeflat
      4 from .defringe import defringe
      5 

~/Applications/miniconda3/envs/dat/lib/python3.9/site-packages/stistools-1.4.1-py3.9.egg/stistools/defringe/mkfringeflat.py in <module>
      2 
      3 from astropy.io import fits
----> 4 from astropy.nddata.utils import block_reduce
      5 import numpy as np
      6 import math

ImportError: cannot import name 'block_reduce' from 'astropy.nddata.utils' (/Users/ldsantos/Applications/miniconda3/envs/dat/lib/python3.9/site-packages/astropy/nddata/utils.py)
```
Maybe we need to update `stistools` to be compatible with `astropy` v5.1?

**Comment by ladsantos** [comment]
Apparently, for v5.1 the correct import statement should be `from astropy.nddata import block_reduce`.

**Comment by ladsantos** [comment]
Ah, never mind, this issue has been corrected in `stistools` v1.4.2.

**Comment by sean-lockwood** [comment]
Duplicate of poorly titled #139.
